lib_test/mirage/unikernel.ml

```diff
@@ -29,27 +29,27 @@ module Server (C: Mirage_console_lwt.S) = struct
   let rec read_all c t =
     VX.read t
     >>= function
-    | Ok `Eof -> OS.Time.sleep_ns (Duration.of_sec 5) >>= fun () -> C.log c "EOF"
-    | Error _ -> OS.Time.sleep_ns (Duration.of_sec 5) >>= fun () -> C.log c "ERR"
+    | Ok `Eof -> Os_xen.Time.sleep_ns (Duration.of_sec 5) >>= fun () -> C.log c "EOF"
```

Comment:
Why is all this code changing to become Xen-specific? Shouldn't this code also run on e.g. Unix?

Reply:
Looking at `build.sh` it looks like the whole unikernel is dedicated to run on xen so I didn't bother using OS-independent modules.

Reply:
I don't have a strong opinion on this (it's just test / example code), but if it's easy to keep it portable that would be nice. If not, don't worry about it...

Reply:
I agree with @talex5 -- we might as well use `OS.Time` and save `Os_xen` for the actual Xen-specific modules.